File: CoursePlanner_Backend/Model/DTOs/CourseRequestDTO.cs
namespace CoursePlanner_Backend.Model.DTOs
{
    public class CourseRequestDTO
    {
        public String Name { get; set; }
        public String Description { get; set; }
        public double Credit_hours { get; set; }
        public String Subject { get; set; }
        public int Course_Number { get; set; }
        public List<int> FeatureIds { get; set; }
        public List<int> CampusIds { get; set; }
        public List<int> PreReqIds { get; set; }
    }
}

File: CoursePlanner_Frontend/src/app/models/CourseRequestDTO.ts
export default interface CourseRequestDTO {
    Name: string,
    Description: string,
    Credit_hours: number,
    Subject: string,
    Course_Number: number,
    PreRequisites: string,
    FeatureIds: Array<number>,
    CampusIds: Array<number>,
    SectionIds: Array<number>
}